### PR TITLE
[AssetMapper] Put importmap in polyfill so it can be hosted locally easily

### LINKED ISSUE #19032: [AssetMapper] Put importmap in polyfill so it can be hosted locally eas…
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#51828
| PR author(s) | @weaverryan
| Merged in    | 6.4

We created this issue to not forget to document this new feature. We would really appreciate if you can help us with this task. If you are not sure how to do it, please ask us and we will help you.

To fix this issue, please create a PR against the 6.4 branch in the [symfony-docs repository](https://github.com/symfony/symfony-docs).

Thank you! :smiley:

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Fix https://github.com/symfony/symfony-docs/issues/19032

Friendly ping @weaverryan, my knowledge on the AssetMapper is super limited 😄

### CODE REVIEW

File: frontend/asset_mapper.rst [hunk 1]
```diff
@@ -1032,14 +1032,20 @@ you expect are being included in the asset map.
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Configure the polyfill for older browsers. Default is `ES module shim`_. You can pass
-any URL to be included, or ``false`` to disable the polyfill.
+an importmap name to load the polyfill, or ``false`` to disable the polyfill loading.
```
Comment (weaverryan) [inline frontend/asset_mapper.rst]:
```suggestion
the key of an item in ``importmap.php`` or ``false`` to disable the polyfill loading.
```

File: frontend/asset_mapper.rst [hunk 2]
```diff
@@ -1032,14 +1032,20 @@ you expect are being included in the asset map.
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Configure the polyfill for older browsers. Default is `ES module shim`_. You can pass
```
Comment (weaverryan) [inline frontend/asset_mapper.rst]:
```suggestion
Configure the polyfill for older browsers. By default, the `ES module shim`_ is loaded via a CDN. You can pass
```

**Review by weaverryan — COMMENTED** [review]
Thanks for this :).

Configuring this will be rare. However, there IS something that will be much more common, and I'm not sure where to mention it. By default, we output a CDN URL to the polyfill. If a user wants to avoid the CDN ad serve the polyfill locally, the only need to run:

```
php bin/console importmap:require es-module-shims
```

No config changes are needed. AssetMapper sees that this is installed locally, and so uses it instead of the CDN. If you have an idea on where we might put that, I'd appreciate it. Even though it doesn't involve actually changing the `importmap_polyfill` config, an easy option would be to mention it right in this section.

**Comment by alexandre-daubois** [comment]
Thank you for the explanation! I added a `tip` section to explain what you explained about the CDN and the command. What do you think? :slightly_smiling_face:

**Review by weaverryan — APPROVED** [review]
👏

**Comment by OskarStark** [comment]
Thank you Alexandre.

**Pull request merged**